### PR TITLE
Restyle for editor tools

### PR DESCRIPTION
**Pull request by wagnerand**
~~Not finished completely, but @muffinresearch wanted to see it. :)~~

### CODE REVIEW

**Comment by muffinresearch** [comment]
Thanks @wagnerand I'll try this out tomorrow and see if I can add it in.

**Comment by wagnerand** [comment]
Fixed syntax error.

**Comment by wagnerand** [comment]
home
queue
review

r?

**Comment by jvillalobos** [comment]
I like.

File: static/css/restyle/restyle.less [hunk 1]
```diff
@@ -1822,3 +1822,231 @@ h3.author .transfer-ownership {
...
+  #navbar ul li.top {
+    width: 20%;
+    border-right: 0;
+    text-align: center;
+  }
+
+  #navbar > ul > li > ul {
+    background-color: #fff;
+    border: none;
+    border-radius: 0 !important;
```
Comment (kewisch) [inline static/css/restyle/restyle.less]: Can you write this rule without using `!important` ?

**Comment by kewisch** [comment]
Is there any old css from before the restyle we can get rid of?

**Comment by kewisch** [comment]
r+ with that considered

File: static/css/restyle/restyle.less (hunk 1 above)
Comment (wagnerand) [inline static/css/restyle/restyle.less]: Probably not. I tried to avoid using `!important` as much as possible. Sometimes, there was just no other way.

**Comment by wagnerand** [comment]
> Is there any old css from before the restyle we can get rid of?
Maybe but I don't want to mess with that. Afaik, we didn't get rid of old styles in other places either.

File: static/css/restyle/restyle.less (hunk 1 above)
Comment (wagnerand) [inline static/css/restyle/restyle.less]: Scratch that. Got rid of them, thank you!

File: static/css/restyle/restyle.less [hunk 2]
```diff
@@ -1822,3 +1822,231 @@ h3.author .transfer-ownership {
   margin-left: 0;
   width: 77%;
 }
+
+/* Editor pages */
+body.editor-tools {
+
```
Comment (EnTeQuAk) [inline static/css/restyle/restyle.less]: Let's not do this whitespace :)

File: static/css/restyle/restyle.less [hunk 3]
```diff
@@ -1822,3 +1822,231 @@ h3.author .transfer-ownership {
...
+    box-shadow: 0 0 4px rgba(0, 0, 0, 0.4);
+  }
+
+  #navbar > ul li > ul li {
+    background-color: #fff;
+    background-image: none;
+    color: @default-font-color;
+    border-bottom: none;
+    text-align: left;
+    a, a:link, a:visited {
```
Comment (EnTeQuAk) [inline static/css/restyle/restyle.less]: Let's put a whitespace above the selectors though :)

File: static/css/restyle/restyle.less [hunk 4]
```diff
@@ -1822,3 +1822,231 @@ h3.author .transfer-ownership {
...
+      color: @default-font-color;
+      border-bottom: none;
+    }
+    a:hover {
```
Comment (EnTeQuAk) [inline static/css/restyle/restyle.less]: Same here?

File: static/css/restyle/restyle.less [hunk 5]
```diff
@@ -1822,3 +1822,231 @@ h3.author .transfer-ownership {
...
+  .editor-stats-table:first-child > div {
+    border-left-width: 0;
+  }
+
+  .editor-stats-table .int {
+    color: @default-font-color;
+  }
+
+  #editors_main, .section {
+
```
Comment (EnTeQuAk) [inline static/css/restyle/restyle.less]: Let's not do this empty line here I think

File: static/css/restyle/restyle.less [hunk 6]
```diff
@@ -1822,3 +1822,231 @@ h3.author .transfer-ownership {
...
+      color: @link-on-white-bg;
+    }
+
+    a:hover {
+      padding-bottom: 5px;
+      border-color: @primary-link;
+    }
+
+    .queue-outer {
+    border-radius: 0;
```
Comment (EnTeQuAk) [inline static/css/restyle/restyle.less]: Identation seems off?

File: static/css/restyle/restyle.less [hunk 7]
```diff
@@ -1822,3 +1822,231 @@ h3.author .transfer-ownership {
...
+
+    #addon.primary {
+
+      .article {
+        border: @detail-border;
+        border-radius: 0;
+      }
+
+      div#review-actions {
+      border: @detail-border;
```
Comment (EnTeQuAk) [inline static/css/restyle/restyle.less]: Indendation seems off again?

**Comment by EnTeQuAk** [comment]
r+wc

Looks pretty cool, let's get the money and let's get feedback from people on this :)

**Comment by wagnerand** [comment]
All fixed. Including @tofumatt so he's in the loop.

**Comment by wagnerand** [comment]
Unrelated intermittent test failure:
_____________________ TestBulkValidationTask.test_validate _____________________
[gw1] linux2 -- Python 2.7.9 /home/travis/build/mozilla/addons-server/.tox/main/bin/python2.7
self = <olympia.zadmin.tests.test_views.TestBulkValidationTask testMethod=test_validate>
   def test_validate(self):
        self.start_validation()
        res = ValidationResult.objects.get()
        self.assertCloseToNow(res.completed)
        assert not res.task_error
        validation = json.loads(res.validation)
        assert res.errors == 1
>       assert validation['messages'][0]['id'] == (
            ['main', 'prepare_package', 'not_found'])
E       AssertionError: assert ['typedetecti...alid_em_type'] == ['main', 'prep..., 'not_found']
E         At index 0 diff: u'typedetection' != 'main'
E         Full diff:
E         - [u'typedetection', u'dictionary_valid_type', u'invalid_em_type']
E         + ['main', 'prepare_package', 'not_found']
res        = <ValidationResult: ValidationResult object>
self       = <olympia.zadmin.tests.test_views.TestBulkValidationTask testMethod=test_validate>
validation = {'compatibility_summary': {'errors': 0, 'notices': 0, 'warnings': 0}, 'detected_type': 'dictionary', 'ending_tier': 5, 'errors': 1, ...}

src/olympia/zadmin/tests/test_views.py:664: AssertionError

cc @diox

**Comment by Noitidart** [comment]
This is utterly beautiful!

**Pull request merged**